### DISCUSSION TITLE
Using CloudFlare Auth service tokens

### DISCUSSION THREAD

**Discussion by ammacdonald3** [Vinzent03/obsidian-git#392] [Q&A]
I have set up Gitea behind a CloudFlare tunnel, which is protected by the CF application authentication.  In the web, I receive a CF login screen before being rerouted to my self-hosted application.  CF also allows users to create service tokens, which can be passed in the header to authenticate before displaying the app.  I've used a similar workflow for the LunaSea app, and I'm wondering if that's also possible with Obsidian-Git in order to access my Gitea repo outside of my LAN.

**Comment by Vinzent03** [comment]
I don't know, because I don't have any experience in that. I'm basically using system installed git on desktop, but I don't know if the authentication flow works as well.
